src/virtiofsd.rs

```diff
@@ -0,0 +1,141 @@
+use std::path::Path;
+use std::path::PathBuf;
+use std::process::Child;
+use std::process::Command;
+use std::process::Stdio;
+use std::sync::LazyLock;
+use std::time::Duration;
+use std::time::Instant;
+
+use anyhow::bail;
+use anyhow::Context as _;
+use anyhow::Result;
+use log::error;
+
+use crate::util::gen_sock;
+
+const VIRTIOFSD_NAME: &str = "virtiofsd";
+static VIRTIOFSD_PATH: LazyLock<Option<PathBuf>> = LazyLock::new(find_virtiofsd_exe);
```

Comment:
`LazyLock` requires Rust 1.80. We can probably work with something else if the bump is not desired.